cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/CatalogController.java

```diff
@@ -28,17 +28,26 @@
 import mil.army.usace.hec.cwms.http.client.ApiConnectionInfo;
 import mil.army.usace.hec.cwms.http.client.HttpRequestBuilder;
 import mil.army.usace.hec.cwms.http.client.HttpRequestResponse;
+import mil.army.usace.hec.cwms.radar.client.model.LocationCatalog;
 import mil.army.usace.hec.cwms.radar.client.model.RadarObjectMapper;
 import mil.army.usace.hec.cwms.radar.client.model.TimeSeriesCatalog;
 
 public final class CatalogController {
 
     private static final String CATALOG_TIMESERIES_ENDPOINT = "catalog/timeseries";
+    private static final String CATALOG_LOCATIONS_ENDPOINT = "catalog/locations";
 
     public TimeSeriesCatalog retrieveTimeSeriesCatalog(ApiConnectionInfo apiConnectionInfo, TimeSeriesCatalogEndpointInput input) throws IOException {
```

Comment:
why do you have location and time series catalogs here, but break out the other catalogs into separate classes?

Reply:
Pretty much just to follow the query schema. I'm open to moving them into the TimeSeriesController and LocationsController classes.